src/super_gradients/common/factories/transforms_factory.py

```diff
@@ -46,6 +47,7 @@ def get(self, conf: Union[str, dict]):
                 if "bbox_params" in conf[_type].keys():
                     bbox_params = conf[_type]["bbox_params"]
                     self._check_bbox_params(bbox_params)
+                    conf = deepcopy(conf)  # Avoid changing the original config.
                     conf[_type]["bbox_params"].update(self.FIXED_BBOX_PARAMS)
```

Comment:
There is a bug due to the fact we modify inplace `conf`: calling `AlbumentationsTransformsFactory` twice on the same config will raise an error inside of `_check_bbox_params`.
Creating a copy prevents this from happening (we don't modify original).